go-controller/pkg/cni/cni.go

```diff
@@ -4,15 +4,17 @@ import (
 	"fmt"
 	"net"
 
-	"github.com/containernetworking/cni/pkg/types/current"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/client-go/kubernetes"
 	corev1listers "k8s.io/client-go/listers/core/v1"
 	"k8s.io/klog/v2"
 	utilnet "k8s.io/utils/net"
 
+	"github.com/containernetworking/cni/pkg/types/current"
```

Comment:
why you move the github.com/containernetworking/cni/pkg/types/current ? I thought the group is 3 party vs project lib